opensearch/src/test/java/org/opensearch/sql/opensearch/client/OpenSearchRestClientTest.java

```diff
@@ -129,6 +130,7 @@ void is_index_exist_with_exception() throws IOException {
   @Test
   void create_index() throws IOException {
     String indexName = "test";
+    setDefaultMediaType(XContentType.JSON);
```

Comment:
Is this expected? How does this work when a new cluster is deployed.

Reply:
Before [8636](https://github.com/opensearch-project/OpenSearch/pull/8636) core used `JSON` hardcoded. Now it uses `DefaultMediaType` which is unset (`null`) by default.
You can re-run these tests on a branch to see the failure.

Reply:
I moved the setDefaultMediaType to a `BeforeAll`. but yes I believe it is the expected Media Type for these tests.

Reply:
@Yury-Fridlyand  I looked into this yesterday, I couldn't figure out the need to introduce a new function for setting default type in core. Probably I should comment in 8636.